src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const SEVEN_DAY_DISCOUNT = 50;
```

Comment:
```suggestion
  const LONG_TERM_DISCOUNT = 50;
```
don't hardcode exact number of days, it may change